.github/workflows/scheduled_release.yml

```diff
@@ -13,21 +13,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Get current date
-        id: date
-        run: echo "name=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
-      - name: Test with environment variables
-        run: echo $TAG_NAME - $RELEASE_NAME
-        env:
-          TAG_NAME: weekly-tag-${{ steps.date.outputs.date }}
-          RELEASE_NAME: weekly-release-${{ steps.date.outputs.date }}
+        run: echo "::set-output name=todays_date::$(date +'%Y-%m-%d')"
       - name: Create Release
         id: create_release
         uses: actions/create-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          tag_name: weekly-tag-${{ steps.date.outputs.date }}
-          release_name: Reekly-release-${{ steps.date.outputs.date }}
+          tag_name: superset-1.0.0-${{ outputs.todays_date.value }}
+          release_name: superset-1.0.0-${{ outputs.todays_date.value }}
```

Comment:
I wish we could read this version number from a shared place such as `setup.py`, then realized we are not doing that. Every release we just manually set a version when preparing the release.

I think there are merits in tracking the version number in Git and putting the canonical current (or next) version number in `setup.py`, like [Airflow](https://github.com/apache/airflow/blob/master/setup.py) does. It helps with auto-release such as this PR and makes it easier to see which version current branch is on.

@villebro @mistercrunch @eschutho what do you think?

Reply:
we could use [pybump](https://github.com/ArieLevs/PyBump) this can help us manage the release version bumps automatically

Reply:
Catching up on the context here, so this is a weekly release, but not the "official" release, correct? More like an unstable weekly build? How will people be using this for example?

Reply:
Same as the nightly/weekly builds of any other softwares---to give developers/users early access to features that aren't officially released (or fully tested) yet. The benefit of checking out a periodic build instead of the `master` branch is that you don't have to run build (`npm install && npm run build`, etc) yourself.

We should probably call the [release script](https://github.com/apache/superset/tree/master/RELEASING) here, too, to make tarballs similar to the official releases.

Reply:
You can try getting the latest release from the tag. Something like `git show-ref latest && git show --pretty=tformat:%d -s latest | grep tag:` will give you a list of all tags on that sha, and you can parse through them to get the one you want. There's an example of this in `https://github.com/apache/superset/blob/master/scripts/tag_latest_release.sh` 

I might also recommend adding something like "weekly-build" or "edge" to the tag, or something so that people don't expect that this release is just a patch on the last one.. there could be breaking changes for example, and we wouldn't be bumping the semantic version on it at all to reflect that.

Reply:
we might even be able to extract just that logic of finding the latest tag version from the script if you want to reuse it.

Reply:
I've added "weekly-build"  and even changed the naming of the workflow
in addition, I added find_latest_tag_version.sh which is kind of a subset of tag_latest_release.sh.
at first, I tried extracting the logic into a "common" script but it was difficult due to bash limitations